### PR TITLE
add logo/icon; update shields

### PR DESCRIPTION
**Pull request by umarcor**
# New Features

* Add logo/icon.

# Changes

* Update shields.

# Bug Fixes

* tbd

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,41 +32,41 @@ if it's a library, framework or CLI tool.
...
 ### Decorators
 
-* [export](https://pyTooling.GitHub.io/pyTooling/Decorators/Visibility.html#export)    
+* [export](https://pyTooling.GitHub.io/pyTooling/Decorators/Visibility.html#export)
   Register the given function or class as publicly accessible in a module.
 
 
 ### Exceptions
 
-* [EnvironmentException](https://pyTooling.GitHub.io/pyTooling/Exceptions/PredefinedExceptions.html#environmentexception)
```
Comment (Paebbels) [inline README.md:48]: The vscode settings are missing to preserve enforced line-breaks.

File: README.md [hunk 2]
```diff
@@ -1,21 +1,22 @@
...
-[![PyPI](https://img.shields.io/pypi/v/pyTooling?logo=PyPI&logoColor=FBE072)](https://pypi.org/project/pyTooling/)
-![PyPI - Python Version](https://img.shields.io/pypi/pyversions/pyTooling?logo=PyPI&logoColor=FBE072)
-![PyPI - Status](https://img.shields.io/pypi/status/pyTooling?logo=PyPI&logoColor=FBE072)
-[![Libraries.io status for latest release](https://img.shields.io/librariesio/release/pypi/pyTooling?logo=librariesdotio)](https://libraries.io/github/pyTooling/pyTooling)
-[![Requires.io](https://img.shields.io/requires/github/pyTooling/pyTooling)](https://requires.io/github/pyTooling/pyTooling/requirements/?branch=main)  
-[![Documentation License](https://img.shields.io/badge/doc%20license-CC--BY%204.0-green?logo=readthedocs)](doc/Doc-License.rst)
-[![Documentation - Read Now!](https://img.shields.io/badge/doc-read%20now%20%E2%9E%9A-blueviolet?logo=readthedocs)](https://pyTooling.GitHub.io/pyTooling)
+[![Sourcecode on GitHub](https://img.shields.io/badge/pyTooling-pyTooling-8cd665.svg?longCache=true&style=flat-square&logo=github&longCache=true&logo=GitHub&labelColor=63bf7f)](https://GitHub.com/pyTooling/pyTooling)
```
Comment (Paebbels) [inline README.md:1]: I'm not sure if the 2 green colors fit well to each other in the same shield (first shield).

image

File: doc/conf.py [hunk 3]
```diff
@@ -80,24 +83,30 @@
...
+	}
+else:
+	html_theme = "alabaster"
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
 
+html_logo = str(Path(html_static_path[0]) / "icon.png")
```
Comment (Paebbels) [inline doc/conf.py:107]: I think the logo is to big in the navigation bar.

image

**Comment by Paebbels** [comment]
@umarcor can we continue with this one?

File: README.md (hunk 1 above)
Comment (umarcor) [reply README.md]: Fixed in https://github.com/pyTooling/pyTooling/pull/38/commits/3e557eb13da0a4f080d61b43d5b312c86ad4d666.

File: README.md (hunk 2 above)
Comment (umarcor) [reply README.md]: Updated to use the default gray on the left side.

File: doc/conf.py (hunk 3 above)
Comment (umarcor) [reply doc/conf.py]: Updated to:

image

**Pull request merged**